mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/HighAvailabilityServicesUtil.java

```diff
@@ -37,12 +38,17 @@
  */
 @Slf4j
 public class HighAvailabilityServicesUtil {
+  private final static AtomicReference<HighAvailabilityServices> HAServiceInstanceRef = new AtomicReference<>();
```

Comment:
Should we move this to some *Registry class?

Reply:
I feel like ideally we should avoid have leaf modules depending on HA service in general but inject the master client properly instead.